uxarray/grid/validation.py

```diff
@@ -65,3 +65,19 @@ def _check_area(self):
     else:
         print("-No face area is close to zero.")
         return True
+
+
+def _check_normalization(grid):
+    """Checks whether all the cartesiain coordinates are normalized."""
+
+    if "node_x" in grid._ds:
+        if not (grid.node_x**2 + grid.node_y**2 + grid.node_z**2).all():
+            return False
+    if "edge_x" in grid._ds:
+        if not (grid.edge_x**2 + grid.edge_y**2 + grid.edge_z**2).all():
+            return False
+    if "face_x" in grid._ds:
+        if not (grid.face_x**2 + grid.face_y**2 + grid.face_z**2).all():
+            return False
+
+    return True
```

Comment:
@hongyuchen1030 

Wondering if this is an appropriate way to check for normalization?

Reply:
Yeah this is the most fundamental way to checked if the coordinates are normalized. However, as I mentioned before, it will run all the nodes, leading to potential performance slow down